libsignal-service/src/groups_v2/manager.rs

```diff
@@ -211,13 +211,13 @@ impl<'a, S: PushService, C: CredentialsCache> GroupsManager<'a, S, C> {
         let password =
             hex::encode(bincode::serialize(&auth_credential_presentation)?);
 
-        Ok(HttpCredentials { username, password })
+        Ok(HttpAuth::Basic { username, password })
     }
 
     pub async fn get_group(
         &mut self,
         group_secret_params: GroupSecretParams,
-        credentials: HttpCredentials,
+        credentials: HttpAuth,
```

Comment:
What's the semantics of setting this to `HttpAuth::None`?

Reply:
see for example [the first use of it here](https://github.com/Michael-F-Bryan/libsignal-service-rs/pull/85/files#diff-78c1e3016616476335a7b4df6a1171673c2af645b0740309febf03a96f35f95aR367)

I suspect that it'll also be useful for structs that are initialized without credentials and need them later (mostly when linking a secondary device for example).

Reply:
I mean the semantics in the `get_group` specifically.